src/rest_framework_dso/serializers.py

```diff
@@ -170,31 +175,33 @@ def many_init(cls, *args, **kwargs):
 
         return list_serializer_class(*args, **list_kwargs)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    @property
+    def fields(self):
```

Comment:
aub niet `fields` overriden. het is veel beter om de fields te herdefinieren in `__init__` (wordt overal gedaan).

Nu wordt deze logica iedere keer uitegevoerd als iemand `self.fields` opgevraagd, wat de request vertraagd.

Reply:
Eigelijk `fields` is een `cached_property` en moet niet met dict overgeschreven worden.
En probleem met cached properties is dat we hebben geen controle over de cache, `self.fields` geroeped minimaal 2 keer:
 - 1 op de moment van de serializer initializatie, waar request is niet altijd aanwezig voor de Nested Serializers
 - 2 op de moment van data serializatie, waar de request is aanwezig, en daarvoor dit kan niet gecached worden.

ref: https://github.com/encode/django-rest-framework/blob/master/rest_framework/serializers.py#L339)